src/Store/Reducer.re

```diff
@@ -53,8 +53,13 @@ let reduce: (State.t, Actions.t) => State.t =
       | WindowFocusGained => {...s, windowIsFocused: true}
       | WindowFocusLost => {...s, windowIsFocused: false}
       | WindowMaximized => {...s, windowIsMaximized: true}
```

Comment:
Is it possible to go from `fullscreen` to `maximized`? If so, the `windowIsFullScreen` value might stay `true` when the `WindowMaximized` event comes in. I couldn't make this happen, though. Maybe we should set `windowIsFullScreen: false` in the `WindowMaximized` case, though?

Reply:
That's a great question -- I didn't consider that! I don't think it's possible, as I couldn't get it to do that either. The second point is correct though -- I'll add that now!

Reply:
This is fixed in the latest commit! Thanks again, @bryphe!